### ISSUE TITLE
Mari0 is already on github

### ISSUE THREAD

**Issue by slambo311** [kazma5290/Mari0-Portable-Ver.-1.6#1]
This is pretty useless, because Mari0's already on github, plus it's already portable.

**Comment by kazma5290** [comment]
Yes it is, and this is a portable app for the program portableapps (Web: portableapps.com). There might not be a point, but it's nice, for people that have portable apps.
